### PR TITLE
Fix display of remaining time in pretrain script

### PR DESCRIPTION
**Pull request by awaelchli**
The remaining time in the pretraining script is printed like:
`remaining time: 35780 days, 3:14:59`

Pretraining happens on the time scale of days and weeks. This should just show X days.
It also cannot really be accurate to the minute, so the added information is only noise.

This PR also logs remaining days now (seconds was useless for the human).

### CODE REVIEW

**Comment by carmocca** [comment]
I disagree here.

This can be useful if you are pertaining small models (like the tinystories models)

It can also be useful to see the progress, `remaining time: 35780 days, 19:14:59` to `remaining time: 35780 days, 3:14:59` makes it easy to see how many hours was reduced.

I would agree with removing seconds, but hh:mm:ss is the clearest way because if you do hh:mm then it might not be clear if it's that or mm:ss for small models

**Comment by carmocca** [comment]
`timedelta` is also smart in that it won't include the days if there's not enough

```python
>>> str(timedelta(seconds=10**4))
'2:46:40'
>>> str(timedelta(seconds=10**8))
'1157 days, 9:46:40'
```

Whereas the current logic would show something like `0.1 days`. Maybe the middle ground is using a datetime format that goes from hh:mm:ss to days only after 24 hours

File: litgpt/pretrain.py [hunk 1]
```diff
@@ -244,15 +243,14 @@ def fit(
                 samples=(state["iter_num"] * train.micro_batch_size),
                 lengths=(state["iter_num"] * train.micro_batch_size * model.max_seq_length),
             )
+            remaining_days = (t1 - total_t0) / (state["iter_num"] - initial_iter) * (max_iters - state["iter_num"]) / 1440
```
Comment (carmocca) [inline litgpt/pretrain.py:246]: Isnt this seconds to days?

```suggestion
            remaining_days = (t1 - total_t0) / (state["iter_num"] - initial_iter) * (max_iters - state["iter_num"]) / 86400
```

**Comment by awaelchli** [comment]
I prefer a simple solution. For short runs where it says 0.1 I dont think it matters.

@rasbt do you want to help break the tie?

**Comment by rasbt** [comment]
I would perhaps not show seconds because it's too much information, but minutes is useful because you can then more easily see if it is making progress or if it is stuck, etc.

But instead of

```
remaining time: 35780 days, 3:14:59
```

how about a more compact

```
remaining time: 35780d, 3h 14m
```

**Comment by rasbt** [comment]
Maybe something we can also merge soon so that we have consistency for the demos?

**Comment by awaelchli** [comment]
I think I'm dropping this one. It's too much effort for so little value. And for the Studios I can make local changes if needed as well.

**Pull request closed without merging**